services/httpd/handler.go

```diff
@@ -319,8 +320,8 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, user *meta.
 
 	// Execute query.
 	w.Header().Add("Connection", "close")
-	w.Header().Add("content-type", "application/json")
-	readonly := r.Method == "GET" || r.Method == "HEAD"
+	w.Header().Add("Content-Type", "application/json")
+	readonly := r.Method == "GET"
```

Comment:
What's wrong with checking for `HEAD`? I know that we don't currently support that for queries, but it really shouldn't harm anything to ensure that, if we do add it, that the query is done in a read-only context.